version-history.md

```diff
@@ -10,6 +10,7 @@ permalink: /version-history/
 
 OpenSearch version | Release highlights | Release date  
 :--- | :--- | :--- 
+[2.6.0](https://github.com/opensearch-project/opensearch-build/blob/main/release-notes/opensearch-release-notes-2.6.0.md) | Includes simple schema for observability, index management UI enhancements, SigV4 authentication in Dashboards, Security analytics enhancements, sentence transformer models for model-serving framework, search backpressure at the coordinator node level, and adding maps to dashboards. Experimental features include a new ML model health dashboard. For a full list of release highlights, see the Release Notes. | 28 February 2023
```

Comment:
Analytics in Security Analytics should be capitalized

Reply:
```suggestion
[2.6.0](https://github.com/opensearch-project/opensearch-build/blob/main/release-notes/opensearch-release-notes-2.6.0.md) | Includes simple schema for observability, index management UI enhancements, Security Analytics enhancements, search backpressure at the coordinator node level, and the ability to add maps to dashboards. Experimental features include a new ML model health dashboard, new text embedding models in ML, and SigV4 authentication in Dashboards. For a full list of release highlights, see the Release Notes. | 28 February 2023
```

Reply:
Change the fragment for ML to "...support for sentence transformer pretrained models in ML..."